tests/upgrade/upgrade_test.go

```diff
@@ -61,12 +63,20 @@ var _ = ginkgo.Describe("[Upgrade]", func() {
 			cChainGenesisStr := network.Genesis.CChainGenesis
 			require.NoError(json.Unmarshal([]byte(cChainGenesisStr), cChainGenesis))
 			unscheduledActivationTime := uint64(upgrade.UnscheduledActivationTime.Unix())
-			cChainGenesis.Config.EtnaTime = &unscheduledActivationTime
+			cChainGenesis.Config.EtnaTimestamp = &unscheduledActivationTime
 			cChainGenesisBytes, err := json.Marshal(cChainGenesis)
 			require.NoError(err)
 			network.Genesis.CChainGenesis = string(cChainGenesisBytes)
```

Comment:
when I was looking at the issue with the upgrade test, seems these lines 
https://github.com/ava-labs/coreth/pull/635/files#diff-3be5ea9687f8af021d9baaa6584d174a7975b181b604e1a759e4ad2e40136912R469-R471
overwrite it with the chainCtx, where is the chainCtx being set for this test? Should we apply the desired behavior there?

Reply:
this is set by the avalanchego flag `upgrade-file-content-key`. It's applied with a flag and then the node passes it to Coreth. We don't actually create chainCtx here.